topology/pynest/hl_api.py

```diff
@@ -2115,3 +2130,33 @@ def PlotKernel(ax, src_nrn, mask, kern=None, mask_color='red',
                              'version of PyTopology')
 
     plt.draw()
+
+
+def SelectNodesByMask(layer, anchor, mask_obj):
+    """
+    Obtain GIDs inside a specified area.
```

Comment:
Please make the description a bit more detailed, i.e., point out that GIDs are obtained inside one topology layer only and that the area is specified by a mask.